utils/spacewalk-utils.changes

```diff
@@ -1,3 +1,4 @@
+- Add EPEL8 for Almalinux 8 and Rocky 8 in spacewalk-common-channels.ini
```

Comment:
But as this is the first change after the last header, the empty line below this is still needed :-)